context-free.md

```diff
@@ -2,21 +2,31 @@
 
 ##  Who is this programming language for?
 
+This programming language is for artists. It allows artists to create complex pattern art with thousands of repeated shapes that might otherwise be tedious and time consuming to create.
+
```

Comment:
Would you say that it is easy for artists to use? If not, do you think it would be for a specific type of artist? It might be good to think about the specific user that would need to use this.